router/broadcast_router.go

```diff
@@ -15,15 +20,21 @@ type broadcastRouterState struct {
 }
 
 func (state *broadcastRouterState) SetRoutees(routees *actor.PIDSet) {
-	state.routees = routees
+	atomic.StorePointer((*unsafe.Pointer)(unsafe.Pointer(&state.routees)), unsafe.Pointer(&routees))
```

Comment:
You should store the value of the parameter `routees`, not its address `&routees` which is a stack address.